corehq/apps/export/views.py

```diff
@@ -2283,6 +2286,56 @@ def get_filters(self, filter_form_data, mobile_user_and_group_slugs):
         return form_filters
 
 
+class DownloadNewSmsExportView(GenericDownloadNewExportMixin, BaseDownloadExportView):
+    urlname = 'new_export_download_sms'
+    page_title = ugettext_noop("Export SMS")
+    form_or_case = None  # todo: remove this property from exports
```

Comment:
I think this should get removed from exports totally and replaced with something like `export_type`, which already exists. Want to dig into this, but was going to separate that out into a different PR